### PR TITLE
Mariadb docs

### PR DESCRIPTION
**Pull request by SamMousa**
Docs to reference the MariaDB extension I created.
Please look at the extension a bit before merging this into the docs!
https://github.com/sam-it/yii2-mariadb

| Q             | A
| ------------- | ---
| Is bugfix?    | no
| New feature?  | yes
| Breaks BC?    | no
| Tests pass?   | yes

### CODE REVIEW

File: docs/guide/start-databases.md [hunk 1]
```diff
@@ -83,6 +85,7 @@ If you need to work with databases support for which isn't bundled with Yii, che
 - [Informix](https://github.com/edgardmessias/yii2-informix)
 - [IBM DB2](https://github.com/edgardmessias/yii2-ibm-db2)
 - [Firebird](https://github.com/edgardmessias/yii2-firebird)
+- [MariaDB](https://github.com/edgardmessias/yii2-mariadb)
```
Comment (cebe) [inline docs/guide/start-databases.md]: should be `sam-it` instead of `edgardmessias`, shouldn't it? ;)

**Comment by cebe** [comment]
how about integrating the functionality into Yii? Not having maria db in the core does not make sense to me, as it is becoming the default mysql installation on most distros.

**Comment by SamMousa** [comment]
I'm fine with it being in the core.

**Comment by SamMousa** [comment]
Fix the copy paste mistake :|

**Comment by samdark** [comment]
Oh, right. I forgot about the fact that Maria is default for many distros out there.

**Comment by cebe** [comment]
Thank you!

**Pull request merged**